api/base.py

```diff
@@ -96,9 +94,12 @@ def __init__(self, request=None, response=None):
         elif drone_request:
             self.superuser_request = True
         else:
-            user = self.app.db.users.find_one({'_id': self.uid}, ['root', 'wheel'])
+            user = self.app.db.users.find_one_and_update({'_id': self.uid}, {'$set': {'lastseen': request_start}}, ['root', 'wheel'])
```

Comment:
Are you sure you want to trigger a DB write on *every request*?
Or maybe I'm misreading this.